src/Rings.jl

```diff
@@ -241,3 +241,40 @@ function is_finite(R::NCRing)
   c == 1 && return true
   throw(NotImplementedError(:is_finite, R))
 end
+
+function _is_noetherian(R::T) where T<:Ring
+  if R isa Union{PolyRing, MPolyRing, LaurentPolyRing, LaurentMPolyRing}
+    return _is_noetherian(coefficient_ring(R))
+  elseif R isa Union{MSeriesRing, SeriesRing}
+    return _is_noetherian(base_ring(R))
+  elseif R isa Union{Integers, <:Field}
+    return true
+  else
+    return false
+  end
+end
+
+@doc raw"""
+    krull_dim(R::Ring)
+
+Return the Krull dimension of the ring $R$. The method is currently only supported for certain commutative Noetherian Rings.
+```jldoctest
+julia> R, x = polynomial_ring(ZZ, [:x]);
+
+julia> krull_dim(R)
+2
+```
+"""
+function krull_dim(R::T) where T<:Ring
+  @req _is_noetherian(R) throw(NotImplementedError(:krull_dim, R))
+  return _krull_dim_noetherian(R)::Int
+end
+
+_krull_dim_noetherian(R::Field) = 0
```

Comment:
A field is always notherian, this should be
```suggestion
krull_dim(R::Field) = 0
```